### DISCUSSION TITLE
作者您好我这边有一个啼笑皆非的问题，不知道是不是我这边模型下载得有问题

### DISCUSSION THREAD

**Discussion by 1461660252** [PaddlePaddle/PaddleOCR#12536] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win10
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：paddlepaddle 2.0.0,paddleocr :2.7.0.3
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):no

中文：目前我是在做一个原神地图识别的相关项目，尝试使用了paddleocr作为我识别文字的工具，发现游戏中有一个地名叫做天工峠，不管是在paddleocr的官网还是我自己的本地，第三个“峠”字永远是识别不对的，所以我选择使用模糊匹配来对原神地图进行匹配，但是倒霉的事情是，原神中还有一个地名叫做天云峡，我目前所使用的模糊匹配算法是fuzzywuzzy里面的fuzzy，应该是 根据字符的该变量来衡量置信度的，但是由于这个天工峠经常被识别成天工峡，就会导致，和天工峠和天云峡这两个不同的地名的置信度是一样的，我应该怎么解决这个问题呢，跪求大佬救命！！！
English:Currently, I am working on a project related to map recognition in the game Genshin Impact, and I have attempted to use PaddleOCR as a tool for recognizing text. However, I have found that there is a location in the game called "天工峠," and the third character "峠" is always recognized incorrectly, both on the PaddleOCR official website and on my local machine. Therefore, I have chosen to use fuzzy matching for Genshin Impact map recognition. Unfortunately, there is another location in the game called "天云峡," and the fuzzy matching algorithm I am currently using is "fuzzy" from the fuzzywuzzy library, which measures confidence based on character variation. However, because "天工峠" is often misrecognized as "天工峡," the confidence levels for these two different locations, "天工峠" and "天云峡," are the same. How can I solve this problem? I desperately need help from an expert!

**Comment by sssjc666** [comment, marked as answer]
ocr默认文本里面应该没有“峠”这个字吧，你可以对照他们的字符文本看看，如果没有的话就去看一下更新文本字符的操作

**Comment by 1461660252** [comment]
好的，我尝试一下
